src/content/quickstart/openshiftgn/_index.md

```diff
@@ -84,6 +84,9 @@ subctl join --kubecontext west --kubeconfig ./merged_kubeconfig broker-info.subm
 subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west
 ```
 
+This will auto-assign Global CIDRs to the clusters. ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.
```

Comment:
```suggestion
BY default this will auto-assign Global CIDRs to the clusters. The ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.
```